bindings/c/test/mako/mako.c

```diff
@@ -783,6 +792,17 @@ int run_workload(FDBTransaction* transaction, mako_args_t* args, int thread_tps,
 			}
 		} /* throttle or txntrace */
 
+		/* enable transaction tagging */
+		if (dotagging > 0) {
+			sprintf(tagstr + KEYPREFIXLEN + TAGPREFIXLENGTH_MAX, "%03d", urand(0, args->txntagging - 1));
```

Comment:
This may put the 3 digit number after a null termination character if prefix is shorter than TAGPREFIXLENGTH_MAX, no?

Reply:
Yes. It will put them after a null termination character. I'd like to consider tags here as 16-byte binary bytes instead of c-string, so I don't regard \x00 as null termination character.

Reply:
Oh, then, you may need to zero-out the buffer first.

Reply:
Done in line 723